ansible/playbooks/roles/repository/files/download-requirements/src/command/dnf_repoquery.py

```diff
@@ -79,14 +92,19 @@ def query(self, packages: List[str], queryformat: str, archlist: List[str], only
         :returns: query result
         """
 
-        def output_handler(output: str):
+        def output_handler(output_stdout: str, output_stderr: str):
             """ In addition to errors, handle missing packages """
-            if not output:
+            if not output_stdout:
                 raise PackageNotfound(f'repoquery failed for packages `{packages}`, reason: some of package(s) not found')
-            elif 'error' in output:
-                raise CriticalError(f'repoquery failed for packages `{packages}`, reason: `{output}`')
+            if 'error' in output_stdout:
+                raise CriticalError(f'Found an error. repoquery failed for packages `{packages}`, reason: `{output_stdout}`')
+            if "Last metadata expiration check" in output_stderr:
```

Comment:
This check does nothing, so not needed? If needed then please, join lines 102-103 together.